TranslatorReasonerAPI.yaml

```diff
@@ -300,16 +325,27 @@ components:
                 $ref: '#/components/schemas/BiolinkRelation'
         relation:
           type: string
-          example: upregulates
-          description: Lower-level relationship type of this edge
+          example: RO:0002447
+          description: >-
+            Query constraint against the relationship type term of this edge,
+            as originally specified by, or curated by inference from, the
+            original external source of knowledge. Note that this should
+            often be specified as predicate ontology term CURIE, although
+            this may not be strictly enforced.
         subject:
           type: string
           example: https://omim.org/entry/603903
-          description: Corresponds to the id of subject node of this edge
+          description: >-
+            Corresponds to the map key CURIE of the
```

Comment:
I don't believe the subject and object need to be CURIEs in the query graph, they could just be identifiers internal to the message. See Patrick's example in his new PR (e.g., 'n0' and 'n1'): https://github.com/NCATSTranslator/ReasonerAPI/pull/175/files.